invenio_rdm_records/services/communities/service.py

```diff
@@ -205,10 +203,22 @@ def _remove(self, identity, community_id, record):
         if community_id not in record.parent.communities.ids:
             raise RecordCommunityMissing(record.id, community_id)
 
-        # check permission here, per community: curator cannot remove another community
-        self.require_permission(
+        # If config is true and there is only 1 communities left to remove
+        is_community_required = current_app.config["RDM_RECORD_ALWAYS_IN_COMMUNITY"]
+        is_last_community = len(record.parent.communities.ids) == 1
+        # Then, check for permissions to remove last community
+        can_remove_last_community = self.check_permission(
```

Comment:
This is checking if the user can remove the community, not really the last one:
```suggestion
        can_remove_community = self.check_permission(
```